__tests__/tests/components/ImageZoom.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { ImageZoom } from 'components';
+
+describe('<ImageZoom />', () => {
+  const defaultProps = {
+    style: { backgroundColor: 'red' },
+    uri: { uri: 'dummy.png' },
+  };
+
+  it('should render clickable image', () => {
+    const wrapper = shallow(<ImageZoom {...defaultProps} />);
+
+    const clickableImg = wrapper.find({ nativeId: 'image-zoom-clickable-img' });
+
+    expect(clickableImg.length).toBe(1);
+  });
+
+  it('should render modal when the user presses Touchable', () => {
```

Comment:
I'd find the element you want to simulate the `press` event on, and the do the check.